ui/src/app/apps/apps.workaround.service.ts

```diff
@@ -0,0 +1,153 @@
+import {Injectable} from '@angular/core';
+import {Observable} from 'rxjs/Observable';
+import {PageRequest} from '../shared/model/pagination/page-request.model';
+import {AppRegistration} from '../shared/model/app-registration.model';
+import {SharedAppsService} from '../shared/services/shared-apps.service';
+import {AppVersion} from '../shared/model/app-version';
+import {Page} from '../shared/model/page';
+import {SortParams} from '../shared/components/sort/sort.component';
+import {Subscriber} from 'rxjs/Subscriber';
+import {AppFilterParams, PaginationParams} from './components/apps.interface';
+
+/**
+ * Service class for the Apps module.
+ * Workaround, to be remove after resolve the related Spring Data Flow Server issues.
+ *
+ * @author Damien Vitrac
+ */
+@Injectable()
+export class AppsWorkaroundService {
+
+  public static cache = {
+    init: false,
+    invalid: true,
+    time: new Date().getTime(),
```

Comment:
I think we should use MomentJs for all our date-time needs. It is already used on the project